src/Features/Core/Portable/CodeLens/CodeLensReferencesService.cs

```diff
@@ -251,7 +251,7 @@ public Task<IEnumerable<ReferenceMethodDescriptor>> FindReferenceMethodsAsync(So
 
                     var result = await Task.WhenAll(descriptorTasks).ConfigureAwait(false);
 
-                    return (IEnumerable<ReferenceMethodDescriptor>)result;
+                    return ((IEnumerable<ReferenceMethodDescriptor>)result).Where(descriptor => descriptor != null);
```

Comment:
:bulb: The `GetMethodDescriptorAsync` method should probably be renamed to `TryGetMethodDescriptorAsync`. The current name suggests that the result will not be `null`.

:bulb:

```csharp
return result.OfType<ReferenceMethodDescriptor>();
```